test/pom.xml

```diff
@@ -39,10 +39,6 @@
 			<artifactId>frankframework-ladybug</artifactId>
 			<scope>provided</scope>
 		</dependency>
-		<dependency><!-- This dependency is here because of the CustomHighDeliveryCountEsbJmsListener -->
```

Comment:
Why was this dependency removed?

Reply:
Deze moet niet weg toch, de andere kan wel weg.